addons/rose/addon/styles/hds/overrides.scss

```diff
@@ -53,3 +53,20 @@ form {
     color: inherit;
   }
 }
+
+// TODO: This is a temporary fix that ensures changing the color theme won't alter the dropdown colors.
+.hds-app-frame__sidebar .hds-side-nav {
```

Comment:
Hds team is helping us find a cleaner solution for this style override. So this fix is just temporary.